pom.xml

```diff
@@ -356,6 +364,12 @@
                 <artifactId>hibernate-validator-annotation-processor</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>${project.groupId}</groupId>
+                <artifactId>hibernate-validator-bom</artifactId>
+                <type>pom</type>
+                <version>${project.version}</version>
+            </dependency>
```

Comment:
Could you also remove dependency management for all public modules (e.g. engine) and use the BOM in integration tests?

That way, we know the BOM is complete :)